src/components/implementation/tests/unit_cbufmalloc/posix_test.c

```diff
@@ -0,0 +1,48 @@
+#include <printc.h>
+#include <cos_component.h>
+#include <cbuf_mgr.h>
+#include <assert.h>
+
+#define NUM 10
+char *mptr[NUM];
+
+void
+cos_init(void *args)
+{
+	int err=0;
+	int i, j;
+
+	for (i = 0; i < NUM; i++) {
+		mptr[i] = (char *)malloc((i+1) * sizeof(int));
+
```

Comment:
I see no reason to have a space between every line here.  Empty lines denote the equivalent of paragraphs.  In no case would the two lines in 
```
a = malloc(...);
assert(a)
```

Be considered separate thoughts.  These are clearly part of the same high-level conceptual operation.